tds/field.go

```diff
@@ -1243,44 +1260,65 @@ func LookupFieldFmt(dataType asetypes.DataType) (FieldFmt, error) {
 	switch dataType {
 	case asetypes.BIGDATETIMEN:
 		f = &BigDateTimeNFieldFmt{}
+		// Based on the default output when string formatting time.Date
+		f.setDisplayMaxLength(34)
 	case asetypes.BIGTIMEN:
 		f = &BigTimeNFieldFmt{}
+		f.setDisplayMaxLength(34)
 	case asetypes.BIT:
 		f = &BitFieldFmt{}
+		f.setDisplayMaxLength(34)
 	case asetypes.DATETIME:
 		f = &DateTimeFieldFmt{}
+		f.setDisplayMaxLength(34)
 	case asetypes.DATE:
 		f = &DateFieldFmt{}
+		f.setDisplayMaxLength(34)
 	case asetypes.SHORTDATE:
 		f = &ShortDateFieldFmt{}
+		f.setDisplayMaxLength(34)
 	case asetypes.FLT4:
 		f = &Flt4FieldFmt{}
 	case asetypes.FLT8:
 		f = &Flt8FieldFmt{}
 	case asetypes.INT1:
 		f = &Int1FieldFmt{}
+		// including sign
+		f.setDisplayMaxLength(4)
```

Comment:
```suggestion
		f.setDisplayMaxLength(3)
```
Actually, `INT1` is an unsigned integer.